resources/wikia/polyfills/bind.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Polyfill for `Function.prototype.bind` because PhantomJS 1.9.8 doens't support it
+ * Should only need to use this for unit tests, as all Wikia's supported browsers support bind.
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind#Polyfill
+ */
+if (!Function.prototype.bind) {
+	Function.prototype.bind = function(oThis) {
```

Comment:
Hm, yeah I could do that. This brings up a question that I have every so often - when you copy and paste a code example, do you clean it up / refactor it to match our coding conventions? I didn't do that for this but now that I'm thinking about it more, I probably should. It's not like this is a versioned library or anything. I'll go ahead and clean this up so it follows our conventions.